test/unit/modules/observer/observer.spec.js

```diff
@@ -356,6 +356,18 @@ describe('Observer', () => {
     })
   })
 
+  it('warn set/delete on non valid values', () => {
+    try {
+      setProp(null, 'foo', 1)
+    } catch (e) {}
+    expect(`Cannot set reactive property on non-object/array value`).toHaveBeenWarned()
```

Comment:
Late to the party, but why use ES6 template literals here (and the next `expect()`)?